specifications/api/swagger.yaml

```
@@ -7544,6 +7544,34 @@ paths:
         - *parametersWalletId
       responses: *responsesDeleteWallet
 
+  /shared-wallets/{walletId}/statistics/utxos:
+    get:
+      operationId: getUTxOsStatisticsShared
+      tags: ["Shared Wallets"]
+      summary: UTxO Statistics
+      description: |
+        <p align="right">status: <strong>stable</strong></p>
+
+        Return the UTxOs distribution across the whole wallet, in the form of a histogram.
```

Just a nitpick:

To me, the terms "UTxO**s** statistics" and "UTxO**s** distribution" feel rather awkward from an English-language perspective.

The terms "UTxO statistics" and "UTxO distribution" are certainly more natural.

If we had no backward-compatibility concerns, I would just do a search-and-replace for:

```patch
- UTxOsStatistics
+UTxOStatistics
```
And others like it.

However, it seems this usage has already been a part of our API specification for a long time, so changing it now would break clients.

Nevertheless, would still suggest fixing the description:
```suggestion
        Return the UTxO distribution across the whole wallet, in the form of a histogram.
```